Simple Incremental/Assets/Scripts/Monobehaviours/HudLevelUI.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(Text))]
+public class HudLevelUI : MonoBehaviour
+{
+    public GameObject player;
+    Text levelText;
+    CharacterLevel characterLevel;
+
+    private void Awake()
+    {
+        characterLevel = player.GetComponent<CharacterLevel>();
+        levelText = GetComponent<Text>();
+        levelText.text = (string)characterLevel.level.ToString();
```

Comment:
We might want to move this to the Start() method or even abstract it out to a separate public method completely (and call it from a GameEventListener on the newly added "Player Level Up" event. 

For now this is ok since we don't have a subscribable object or event to listen to, but I predict some bugs down the road with different monobehaviours running Awake in different orders each run.